drivers/xaudio2/audio_driver_xaudio2.cpp

```diff
@@ -97,7 +97,7 @@ void AudioDriverXAudio2::thread_func(void *p_udata) {
 
 	AudioDriverXAudio2 *ad = (AudioDriverXAudio2 *)p_udata;
 
-	uint64_t usdelay = (ad->buffer_size / float(ad->mix_rate)) * 1000000;
+	//uint64_t usdelay = (ad->buffer_size / float(ad->mix_rate)) * 1000000;
```

Comment:
If it's not used it can be deleted. CC @marcelofg55

Reply:
I didn't write that, but seems like it can be removed safely.

Reply:
Thanks. I've updated the pull request.